test-fork/peripheral/grants/AuraArbBalGrant.spec.ts

```diff
@@ -0,0 +1,276 @@
+import { expect } from "chai";
+import { BigNumber, BigNumberish, ethers } from "ethers";
+import hre, { network } from "hardhat";
+
+import {
+    Account,
+    AuraArbBalGrant,
+    AuraArbBalGrant__factory,
+    IBalancerPool,
+    IBalancerPool__factory,
+    IBalancerVault__factory,
+    IERC20,
+    IERC20__factory,
+    IWeightedPoolFactoryV2__factory,
+} from "../../../types";
+import { config } from "../../../tasks/deploy/mainnet-config";
+import { getPoolAddress, Phase2Deployed } from "../../../scripts/deploySystem";
+import { impersonateAccount, ZERO_ADDRESS, simpleToExactAmount } from "../../../test-utils";
+import { AssetHelpers } from "@balancer-labs/balancer-js";
+
+const FORK_BLOCK = 17274000;
+const ARB_TOKEN = "0xB50721BCf8d664c30412Cfbc6cf7a15145234ad1";
```

Comment:
Looks like this is the arb token on ethereum but I thought the grant was intended to be on arbitrum?

On arbitrum token is `0x912ce59144191c1204e64559fe8253a0e49e6548` instead

Reply:
Yeah this fork test just runs against ETH, last time I checked arb alchemy endpoint had some issues forking, but that was a while ago so maybe it's resolved. Either way.

Reply:
can always fork test against the ankr public rpc aggregator (I tend to use these tbh): https://arbitrum.public-rpc.com/